src/olympia/addons/indexers.py

```diff
@@ -32,7 +32,7 @@ def get_mapping(cls):
                 'properties': {
                     'id': {'type': 'long'},
 
-                    'app': {'type': 'long'},
+                    'app': {'type': 'byte'},
```

Comment:
The same field is a `byte` elsewhere, and in 2.x field definitions need to be consistent across different document types on the same index (yes, it's annoying). On the plus side, `app` should really be a `byte`, there is no reason to use a `long`.